posthog/temporal/common/worker.py

```diff
@@ -23,6 +23,51 @@ def _debug_pyarrows():
         pa.log_memory_allocations(enable=True)
 
 
+async def create_worker(
+    host: str,
+    port: int,
+    metrics_port: int,
+    namespace: str,
+    task_queue: str,
+    workflows: collections.abc.Sequence[type],
+    activities,
+    server_root_ca_cert: str | None = None,
+    client_cert: str | None = None,
+    client_key: str | None = None,
+    graceful_shutdown_timeout: dt.timedelta | None = None,
+    max_concurrent_workflow_tasks: int | None = None,
+    max_concurrent_activities: int | None = None,
+) -> Worker:
+    """Connect to Temporal server and return a Worker."""
```

Comment:
I noticed we're not keeping `_debug_pyarrows()` here. I'm not actually sure if it's still useful (I think Tom said we can turn off the debug logging anyway now we've resolved the memory leak issue)

Reply:
Hmm, I can check if it's still needed.

Reply:
I've removed them.